prow/plugins/label/label_test.go

```diff
@@ -108,6 +108,15 @@ func TestLabel(t *testing.T) {
 			expectedRemovedLabels: []string{},
 			commenter:             orgMember,
 		},
+		{
+			name:                  "Add Single Triage Label",
+			body:                  "/triage need-information",
+			repoLabels:            []string{"area/infra", "triage/need-information"},
+			issueLabels:           []string{"area/infra"},
+			expectedNewLabels:     formatLabels("triage/need-information"),
```

Comment:
@cblecker hi, I just noticed that the label we already have is `close/needs-information` (which I have PR created to change to `triage/needs-information`). Since this is a test so I guess any label name is fine but might be good idea to use `needs-information` instead of `need-information`? Thanks!

Reply:
yup, can fix.

Reply:
@cblecker Thanks!!